libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -118,6 +118,13 @@ const AP_Param::GroupInfo AP_MotorsUGV::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THST_ASYM", 14, AP_MotorsUGV, _thrust_asymmetry, 1.0f),
 
+    // @Param: REV_DELAY
+    // @DisplayName: Motor reversal delay
+    // @Description: For reversible motors that need a delay before they can reverse. When greater than zero the throttle will go to zero for this amount of time before outputting the new throttle.
```

Comment:
I've no suggestions for wording, but I think it needs to be clear that is the motor reversing direction not just the vehicle going into reverse. That is to say, the delay will still apply when going from vehicle reverse to vehicle forwards.

Reply:
fixed